libmamba/tests/src/core/test_util_string.cpp

```diff
@@ -385,6 +385,22 @@ namespace mamba
         {
             CHECK_EQ(concat("aa", std::string("bb"), std::string_view("cc"), 'd'), "aabbccd");
         }
+
+        TEST_CASE("get_common_substr")
+        {
+            CHECK_EQ(get_common_substr("", ""), "");
+            CHECK_EQ(get_common_substr("", "test"), "");
+            CHECK_EQ(get_common_substr("test", "test"), "test");
+            CHECK_EQ(get_common_substr("test/chan", "test/chan"), "test/chan");
```

Comment:
Am I getting this correct?
```suggestion
            CHECK_EQ(get_common_substr("test/chan", "test/chan"), "test/chan");
            CHECK_EQ(get_common_substr("st/ch", "test/chan"), "");
            CHECK_EQ(get_common_substr("test/ch", "test/chan"), "test");
```

Reply:
Good catch! I didn't think about this because I was focused on the use case but I added more constraints so that the tests pass, thanks!